version: '3'
services:
  sync:
    image: wazoplatform/wait
    environment:
      TIMEOUT: "${INTEGRATION_TEST_TIMEOUT}"

  confgend:
    image: wazoplatform/wazo-confgend-tests
    ports:
      - 8669
    volumes:
      - "../..:/usr/local/src/wazo-confgend"
      - "./etc/wazo-confgend/conf.d/50-default.yml:/etc/wazo-confgend/conf.d/50-default.yml"
      # - "${LOCAL_GIT_REPOS}/xivo-dao/xivo_dao:/opt/venv/lib/python3.7/site-packages/xivo_dao"
      # - "${LOCAL_GIT_REPOS}/xivo-lib-python/xivo:/opt/venv/lib/python3.7/site-packages/xivo"

  postgres:
    image: wazoplatform/wazo-confd-db-test:local
    ports:
      - "5432"
